README.md

```diff
@@ -40,6 +38,28 @@ npm install express-openid-connect
 
 Follow our [Secure Local Development guide](https://auth0.com/docs/libraries/secure-local-development) to ensure that applications using this library are running over secure channels (HTTPS URLs). Applications using this library without HTTPS may experience "invalid state" errors.
 
+### Configure Auth0
+
+Create a **Regular Web Application** in the [Auth0 Dashboard](https://manage.auth0.com/#/applications).
+
+> **If you're using an existing application**, verify that you have configured the following settings in your Single Page Application:
```

Comment:
It looks like this is copy pasted from a SPA, we should remove the reference to SPA but also ensure the configuration below is correct for a Regular Web App, which might be different from a SPA.

Reply:
I'd verified this using the sample, just overlooked the SPA reference here, will fix